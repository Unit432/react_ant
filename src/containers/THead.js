import { connect } from 'react-redux'
import { toggleSideMenu } from '../actions'
import TopHead from '../components/TopHead'

const mapDispatchToProps = dispatch => ({
  toggleSideMenu: () => dispatch(toggleSideMenu)
})

const THead = connect(null, mapDispatchToProps)(TopHead)

export default THead
