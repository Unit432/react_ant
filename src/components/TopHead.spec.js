import React from 'react'
import THead from './TopHead'
import renderer from 'react-test-renderer'

describe("HeadMenu", () => {
  it('HeadMenu renders a toggle icon', () => {
    const rendered_head = renderer
      .create(<THead />)
      .toJSON();
    expect(rendered_head).toMatchSnapshot()
  })
})
