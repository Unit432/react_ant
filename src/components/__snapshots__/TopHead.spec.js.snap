// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeadMenu HeadMenu renders a toggle icon 1`] = `
<div
  className="ant-layout-header"
  style={
    Object {
      "background": "#fff",
      "padding": 0,
    }
  }
>
  <i
    className="anticon anticon-menu-unfold trigger"
    onClick={undefined}
  />
</div>
`;
